runelite-client/src/main/java/net/runelite/client/plugins/statusbars/Viewport.java

```diff
@@ -34,16 +34,17 @@
 enum Viewport
 {
 	RESIZED_BOX(WidgetInfo.RESIZABLE_VIEWPORT_OLD_SCHOOL_BOX, WidgetInfo.RESIZABLE_VIEWPORT_INTERFACE_CONTAINER,
-			new Point(20, -4), new Point(0, -4)),
+			new Point(20, -4), new Point(0, -4), true),
 	RESIZED_BOTTOM(WidgetInfo.RESIZABLE_VIEWPORT_BOTTOM_LINE, WidgetInfo.RESIZABLE_VIEWPORT_BOTTOM_LINE_INTERFACE_CONTAINER,
-			new Point(61, 8), new Point(35, -12)),
+			new Point(61, 8), new Point(35, -12), true),
 	FIXED(WidgetInfo.FIXED_VIEWPORT, WidgetInfo.FIXED_VIEWPORT_INTERFACE_CONTAINER,
-			new Point(20, -4), new Point(0, -4)),
+			new Point(20, -4), new Point(0, -4), false),
 	FIXED_BANK(WidgetInfo.BANK_CONTAINER, WidgetInfo.BANK_INVENTORY_ITEMS_CONTAINER,
-			new Point(20, -4), new Point(0, -4));
+			new Point(20, -4), new Point(0, -4), false);
 
 	private WidgetInfo container;
 	private WidgetInfo viewport;
 	private Point offsetLeft;
 	private Point offsetRight;
+	private boolean variableWidth;
```

Comment:
It looks like this isn't used? Seems like this should be used to prevent bars drawn on the inventory UI from being drawn at variable sizes. (they'd look bad if they were larger or smaller than they currently are)

Reply:
I'll check but I tested the plugin and wrote code so that it only affects the width in resizeable. The bar width uses the DEFAULT_WIDTH when it's not resizeable.

Reply:
It looks like it does only affect resizable (see the changes in StatusBarsOverlay), but this added field doesn't cause that behavior. One or the other should be removed/changed.

Reply:
oh I see it now sorry yeh I tried that way realized it wouldn't work and forgot about the variable I created

Reply:
should be good now @Nightfirecat